CHANGELOG.md

```diff
@@ -4,6 +4,28 @@
 
 - "You miss 100 percent of the chances you don't take. — Wayne Gretzky" — Michael Scott
 
+- **feat(solid): Remove need to pass router hooks to solid integration**
+
+This release introduces breaking changes to the `@sentry/solid` package.
```

Comment:
```suggestion
This release introduces breaking changes to the `@sentry/solid` package (which is currently out in alpha).
```

Maybe something like this to make clear that this package is alpha? Just so we do not get other people confused why we are shipping a breaking change in a minor release 😅